spec/lib/clearbit/discovery_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe Clearbit::Discovery do
+  before do |example|
+    Clearbit.key = 'clearbit_key'
+  end
+
+  context 'Discovery API' do
+    it 'should call out to the Discovery API' do
```

Comment:
can we use [Imperative mood](https://en.wikipedia.org/wiki/Imperative_mood) instead of `should` for these?

```ruby
it 'returns results from the Discovery API' do
  # ...
end
```